order/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+# Create your models here.
+class Order_item(models.Model):
```

Comment:
```suggestion
class OrderItem(models.Model):
```
클래스 네이밍 컨벤션은 이렇게 하셔야 합니다.

Reply:
넵!!